Community/manage_dhcp/manage_dhcp_config.py

```diff
@@ -0,0 +1,7 @@
+# Copyright 2019 BlueCat Networks. All rights reserved.
+
+# Default Configuration ID
+DEFAULT_CONFIG_NAME = 'Default'
```

Comment:
Default configuration and default view already exist in Gateway's general configurations, it there a reason why it is in a separate file here?

Reply:
It's something I ended up doing for some of my workflows. I believe it was to have all the settings in one place, rather than across multiple sources. It also means you could have a different view for different workflows.